core/src/main/java/org/bitcoinj/wallet/DefaultKeyChainFactory.java

```diff
@@ -72,7 +72,7 @@ public DeterministicKeyChain makeSpendingFriendKeyChain(Protos.Key key, Protos.K
         if (isMarried)
             throw new UnsupportedOperationException("Married Friend Keychains are not allowed");
         else if(accountPath.get(0).equals(ChildNumber.NINE_HARDENED) && /* allow any coin type */
-                accountPath.get(2).equals(ChildNumber.FIVE_HARDENED) && accountPath.get(3).equals(ChildNumber.ONE_HARDENED))
+                accountPath.get(2).equals(new ChildNumber(15, true)))
```

Comment:
What `15` means in this context? Since we use it more than once, can we store it in a variable?

Reply:
Turns out that there a variable, but I forgot about it.

15 means DashPay Feature, while 5 is for Identities.  These numbers are defined in the DashPay Spec and related documents.

https://github.com/dashevo/dashj/pull/90/files/21b0c733271f42d519a373471fb0b4729efb6413#diff-e596d3fab8d7016ce7f629ee8ae93db5R13

Reply:
Fixed here:  https://github.com/dashevo/dashj/pull/90/commits/dd8a7bffac044642ba1b90710f012cf314a2e98e